tests/testStreamClosedMethods.cpp

```diff
@@ -211,11 +211,12 @@ TEST_F(StreamClosedReturnValues, GetFramesWrittenReturnsLastKnownValue){
     mBuilder.setFormat(AudioFormat::I16);
     mBuilder.setChannelCount(1);
     ASSERT_TRUE(openStream());
+    ASSERT_EQ(mStream->setBufferSizeInFrames(mStream->getBufferCapacityInFrames()), Result::OK);
```

Comment:
Without setting setBufferSizeInFrames, OpenSL ES has a bufferSize of 0 so write doesn't work

Reply:
Created separate PR #1582 to fix this code bug. This PR will be too noisy otherwise.